spec/features/return_applications/return_part_payment_applications_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'rails_helper'
+
+RSpec.feature 'When part-payment applications are returned', type: :feature do
+  include Warden::Test::Helpers
+  Warden.test_mode!
+
+  let(:office) { create :office }
+  let(:user) { create :user, office: office }
+
+  let(:application1) { create :application_full_remission, office: office }
+  let!(:evidence1) { create :part_payment, application: application1 }
+  let(:application2) { create :application_full_remission, office: office }
+  let!(:evidence2) { create :part_payment, application: application2 }
+
+  before { login_as user }
+
+  context 'when on home page' do
+
+    before { visit root_path }
+
+    scenario 'shows the applications that are waiting for part-payment' do
+      within '.waiting-for-part_payment' do
+        expect(page).to have_content(application1.reference)
+        expect(page).to have_content(application2.reference)
+      end
+    end
+
+    context 'processing an application' do
+      before { click_link application1.reference }
+      scenario 'shows the application data' do
+        expect(page).to have_content 'Process part-payment'
+        expect(page).to have_content application1.applicant.full_name
+      end
+
+      scenario 'shows "Start now" button' do
```

Comment:
I would join this scenario with the previous one and the next one as well. I know that generally it's a good practice to have only one assertion per text, but all these only assert the page content. There's no action which would make the page different and running 3 scenarios is unnecessarily slow, I think.